bindings/python/tests/tenant_tests.py

```diff
@@ -65,11 +66,11 @@ def test_tenant_operations(db):
 
     t1_entry = tenant_list[0].value
     t1_json = json.loads(t1_entry)
-    p1 = t1_json['prefix'].encode('utf8')
+    p1 = base64.b64decode(t1_json['prefix']['base64'])
```

Comment:
Would it be worth adding a check that the printable field is also formatted correctly and corresponds to the base64 value?

Reply:
We don't really have an easy way to do this in Python, but this check is done in simulation

Reply:
To elaborate, the reason this is difficult in Python is that I don't believe we have an implementation of `printable` (or `unprintable`) such that we could easily compare the binary string with the encoded printable one. We could add such an implementation, but I figured the simulated check of this property would be sufficient.